# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Parent  050581ceb3b2cb497681851bd953eea49fc6e3f9
Branding 1: Adjust to bug 1892849: Copy addressbook.ico as well.

diff --git a/mail/branding/thunderbird/addressbook.ico b/mail/branding/betterbird/addressbook.ico
copy from mail/branding/thunderbird/addressbook.ico
copy to mail/branding/betterbird/addressbook.ico
